package.json

```diff
@@ -38,11 +38,13 @@
     "test": "jest"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
-    "babel-plugin-syntax-jsx": "^6.18.0",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-syntax-jsx": "^7.0.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.0.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Let's remove this `babel-core`. It does not seem to have any usage.

Reply:
babel-jest uses it